src/main/java/use_cases/GameCreationResponseModel.java

```diff
@@ -0,0 +1,24 @@
+package use_cases;
+
+/**
+ * A simple repsonse model. It contains a boolean on whether a Game was created or not when a User attempts to create a Game.
+ */
+public class GameCreationResponseModel {
+    private boolean gameCreated;
+
+    /**
+     * Create a GameCreationResponseModel with the given parameter.
+     * @param created True iff a Game was created in the input boundary/interactor.
+     */
+    public GameCreationResponseModel(boolean created) {
+        this.gameCreated = created;
+    }
+
+    /**
+     * Get whether a Game was created.
+     * @return True iff a Game was created.
+     */
+    public boolean getCreated() {
```

Comment:
getGameCreated might be a better method name here since the variable we are returning is gameCreated.